test/JDBC/input/views/column_domain_usage-vu-prepare.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TYPE typ1 FROM char(32) NOT NULL;
```

Comment:
correct but for prepare and verify script, we can't give different name based on test name as it will create problem in calling and dropping the objects although given the appropriate unique name to all.

Reply:
I meant use the unique name based on file where object is created, that is simple to follow. So xyz_vu_prepare_test

Reply:
updated!